torch/csrc/jit/passes/quantization.cpp

```diff
@@ -986,10 +987,25 @@ void InsertObserversHelper::preprocess(
   }
 }
 
-// Returns true if the value is the weight to LSTM operator.
-bool isDynamicLSTMWeight(Value* v, Use use, bool is_dynamic) {
-  return is_dynamic && use.user->kind() == Symbol::aten("lstm") &&
-      (use.offset == 2);
+bool useQuantizable(Value* v, Use use, bool is_dynamic) {
+  Node* n = use.user;
+
+  const AtenFuncArgs& aten_func_args = AtenFuncArgs({{"lstm", 2}});
+  const CallFuncArgs& call_func_args = CallFuncArgs({{"batch_norm", 1}});
+  for (const auto& func_arg : aten_func_args) {
+    if (n->kind() == Symbol::aten(func_arg.func_name)) {
+      return v == n->inputs().at(func_arg.arg_index);
+    }
+  }
+
+  for (const auto& func_arg : call_func_args) {
+    if (n->kind() == prim::CallFunction &&
+        getFuncName(n->inputs()[0]) == func_arg.func_name) {
+      return v == n->inputs().at(func_arg.arg_index);
+    }
+  }
```

Comment:
I think you can use isAtenFuncNthArg and isCallFunctionNthArg here, also you can refactor these functions to not take value as input, and just compare `use.offset` with the `arg_index`

Reply:
the logic seems a bit different here actually, and this is due to the convoluted check logic that we fall back to nodeQuantizable when the check fails. Can we add a default index for aten and call function in nodes in nodeQuantizable?